verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -59,6 +61,10 @@ interface TableRow {
     name: string;
     // origin
     url: string;
+    // automation
+    // schedule
```

Comment:
nit: should //schedule go below awsRuleArn?

Reply:
Both of these fields are in automation.schedule. Sorry if this is a strange way to denote that -- I'll change the comment format.